sanitizers/src/test/java/com/example/ClassLoaderLoadClass.java

```diff
@@ -22,9 +22,11 @@ public static void fuzzerTestOneInput(FuzzedDataProvider data) throws Interrupte
     String input = data.consumeRemainingAsAsciiString();
     try {
       // create an instance to trigger class initialization
-      ClassLoaderLoadClass.class.getClassLoader().loadClass(input).getConstructor().newInstance();
-    } catch (ClassNotFoundException | InvocationTargetException | InstantiationException
-        | IllegalAccessException | NoSuchMethodException ignored) {
+      ClassLoaderLoadClass.class.getClassLoader().loadClass(input).newInstance();
+      // TODO(khaled): this fails to reproduce the finding. It seems that this is related not
```

Comment:
I don't quite understand this comment. If we don't catch all ways a constructor may end up being called, isn't that a sign that something is off with the new (old) approach?

Reply:
I've reverted this change since it is not used now by the tests for disabled hooks. However, I think the issue is not related to the `jaz.Zer` class per se. If the tested code caches the exceptions we throw and silently handles them, we will not observe a test failure when running in the regression test mode. On the other hand, we throw 
a hard-to-catch error and report a finding. The `FuzzerSecurityIssue[Low|Medium|High|Critical]` exceptions can be handled differently in fuzzing and regression modes.

Reply:
I see, that's a problem with our test setup then, right? With JUnit, findings reported by hooks can't be swallowed even in regression test mode since `reportFindingFromHook` stores them in a global variable.